NAudio/Wave/WaveOutputs/WasapiOutRT.cs

```diff
@@ -450,6 +450,11 @@ public PlaybackState PlaybackState
             }
         }
 
+        /// <summary>
+        /// Volume (not implemented)
+        /// </summary>
+        float IWavePlayer.Volume { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
Get should return 1.0f, Set should throw a `NotSupportedException`

Reply:
Agreed, fixed.